src/app/market/buy/checkout-process/checkout-process.component.ts

```diff
@@ -218,6 +218,13 @@ export class CheckoutProcessComponent implements OnInit, OnDestroy {
     this.getCache();
   }
 
+  clearForm(): void {
+    if (Object.keys(this.selectedAddress).length > 0) {
+      this.selectedCountry = undefined;
```

Comment:
Does this need to be explicitly cleared? Is it not cleared as part of clearing the form?

Reply:
no it requires explicitly clearing